### DISCUSSION TITLE
set default label color for app?

### DISCUSSION THREAD

**Discussion by wyhinton** [fltk-rs/fltk-rs#789] [Q&A] [answered]
Is it possible to set the default label color for an app?

Or do we have to use ```my_widg.set_label_color(Color::White)``` for all of our widgets?

**Comment by MoAlyousef** [comment, marked as answer]
I'm not sure there is a way to set the default label color in FLTK. Anyways fltk-rs doesn't expose any such method.

**Reply by MoAlyousef** [reply]
Sorry I totally forgot about app::foreground, which like app::background takes an RGB value, it should change the default color of labels.
